src/styles/index.scss

```diff
@@ -1,3 +1,74 @@
+$sell-size: 100px;
```

Comment:
The variable name '$sell-size' appears to be a typo. It is likely intended to be '$cell-size', as it represents the size of each calendar cell.